eth-connector/tests/fungible_token_tests.rs

```diff
@@ -260,28 +258,77 @@ async fn test_storage_balance_bounds() {
 }
 
 #[tokio::test]
-async fn test_set_paused_flags() {
+async fn test_pa_pause_feature() {
     let contract = deploy_and_init().await.unwrap();
     contract
-        .set_paused_flags(UNPAUSE_ALL)
+        .pa_pause_feature("withdraw".to_string())
         .max_gas()
         .transact()
         .await
         .unwrap();
```

Comment:
As I remember, to unwrap is not enough to check that everything is ok. The `ExecutionFinalResult` has methods `is_success` and `is_failure`. I would add an assertion that the result of the transaction is successful like this.
```rust
 let result = contract
         .pa_pause_feature("withdraw".to_string())
         .max_gas()
         .transact()
         .await
         .unwrap();
assert!(result.is_success());
```

Reply:
fixed: https://github.com/aurora-is-near/aurora-workspace/pull/39/commits/c1ff238262c6783c33d1cd04a9b9b2a51f52469b